src/providers/twitch/TwitchChannel.cpp

```diff
@@ -739,22 +739,22 @@ void TwitchChannel::refreshBadges()
 
 void TwitchChannel::refreshCheerEmotes()
 {
-    /*auto url = Url{"https://api.twitch.tv/kraken/bits/actions?channel_id=" +
-                   this->getRoomId()};
-    auto request = NetworkRequest::twitchRequest(url.string);
-    request.setCaller(QThread::currentThread());
-
-    request.onSuccess(
-        [this, weak = weakOf<Channel>(this)](auto result) -> Outcome {
+    QString url("https://api.twitch.tv/kraken/bits/actions?channel_id=" +
+                this->roomId());
+    NetworkRequest::twitchRequest(url)
+        .onSuccess([this,
+                    weak = weakOf<Channel>(this)](auto result) -> Outcome {
             auto cheerEmoteSets = ParseCheermoteSets(result.parseRapidJson());
             std::vector<CheerEmoteSet> emoteSets;
 
-            for (auto &set : cheerEmoteSets) {
+            for (auto &set : cheerEmoteSets)
+            {
                 auto cheerEmoteSet = CheerEmoteSet();
                 cheerEmoteSet.regex = QRegularExpression(
                     "^" + set.prefix.toLower() + "([1-9][0-9]*)$");
```

Comment:
QRegularExpression::CaseInsensitiveOption should be used here instead of making it lowercase